client/tests/faucet.ts

```diff
@@ -1,36 +1,62 @@
 import { type Frame, type FullConfig, type Locator, type Page, expect, test } from "@playwright/test";
+import { stringToHex } from "@polkadot/util";
 
 type FormSubmit = {
   address: string;
-  recaptcha: string;
+  captchaResponse: string;
   parachain_id?: string;
 };
 
-const getFormElements = async (page: Page, getCaptcha = false) => {
+type CaptchaProvider = "recaptcha" | "procaptcha";
+
+const getFormElements = async (page: Page, captchaProvider: "recaptcha" | "procaptcha", getCaptcha = false) => {
```

Comment:
Just so we can use the type declared above

```suggestion
type CaptchaProvider = "recaptcha" | "procaptcha";

const getFormElements = async (page: Page, captchaProvider: CaptchaProvider, getCaptcha = false) => {
```

Reply:
Wow, good point :man_facepalming:

Reply:
Hey @Bullrich do you have any shared code/types between client and the backend? Both backend and frontend need to share this type and a getter for it to be able to cast from .env `string` type to `CaptchaProvider` type.

e.g.

```typescript
// provider comes from .env file so is string type
export const getCaptchaProvider = (provider: string): CaptchaProvider => {
  if (provider === CaptchaProvider.procaptcha) return CaptchaProvider.procaptcha;
  if (provider === CaptchaProvider.recaptcha) return CaptchaProvider.recaptcha;
  throw new Error(`⭕ - Invalid captcha provider: ${provider}`);
};
```

Reply:
Unfortunately we don't have any.

There are a couple of duplicated clases/objects, but it didn't make sense to take the work to unify everything. 

For example, the `networkData` object exists both in `src/networkData.ts` and `client/src/lib/utils/networkData.ts`.

Feel free to duplicate the code